src/info/worker.ts

```diff
@@ -683,6 +616,23 @@ function activeGuidance(segments: SubSegment[][], tSegmentDuration: number,
 
     // we're not done with the current subsegment
     else {
+      
+            // get the distance from where we currently are
+            const coord = currentSubSegment.coordinates[currentSubSegmentPointIndex];
+            const setPoint = new Vector(coord.x, coord.y);
+            const diff = setPoint.subtract(convPosEE.clone());
+
+            // don't set the time-based guidance until we have reached the first position
+            // set dist threshold
+            if (currentSubSegmentPointIndex == 0 && diff.mag() >= threshold) {
```

Comment:
threshold is undefined?

Reply:
this should be defined at the top now as a const